it/mail/chrome/messenger/preferences/advanced.dtd

```diff
@@ -62,42 +62,42 @@
 <!ENTITY showConnections.label         "Connessione...">
 <!ENTITY showConnections.accesskey     "n">
 
-<!ENTITY offlineCompact.label "Compatta le cartelle solo se si guadagnano più di ">
+<!ENTITY offlineCompact.label "Comprimi le cartelle se lo spazio risparmiato è maggiore di ">
 <!ENTITY offlineCompact.accesskey "C">
 <!ENTITY kb.label "kB">
 
 <!-- Certificates -->
 <!ENTITY certSelection.label "Quando un server richiede il mio certificato:">
-<!ENTITY certs.auto                      "Selezionane uno automaticamente">
+<!ENTITY certs.auto                      "Seleziona certificato automaticamente">
 <!ENTITY certs.ask                       "Chiedi ogni volta">
 
-<!ENTITY viewCertificates.label         "Mostra certificati">
+<!ENTITY viewCertificates.label         "visualizza certificati">
 <!ENTITY viewCertificates.accesskey     "M">
 <!ENTITY viewCRLs.label                 "Elenchi di revoca">
 <!ENTITY viewCRLs.accesskey             "E">
-<!ENTITY viewOCSP.label                 "Verification">
+<!ENTITY viewOCSP.label                 "Verifica">
 <!ENTITY viewOCSP.accesskey             "f">
 <!ENTITY viewSecurityDevices.label      "Dispositivi di sicurezza">
 <!ENTITY viewSecurityDevices.accesskey  "D">
 
 <!-- Web Services -->
-<!ENTITY webservicesInfo.label          "Configura i servizi web utilizzati all'interno Postbox.">
+<!ENTITY webservicesInfo.label          "Configura i servizi web utilizzati all'interno di Postbox.">
 
-<!ENTITY textStartUp   "Quando si avvia:">
+<!ENTITY textStartUp   "Durante l'avvio:">
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+<!ENTITY textStartUp   "All'avvio:">